app/requirements.txt

```
@@ -34,8 +34,9 @@ pytz==2018.4
 requests==2.21.0
 six==1.11.0
 seqeval==0.0.6
-social-auth-app-django==3.1.0
-social-auth-core[azuread]==3.0.0
+social-auth-app-django==4.0.0
+social-auth-core[azuread]==3.3.3
+social-auth-core[openidconnect]==3.3.3
```

To my understanding, the canonical way to specify multiple extra dependencies would be like this:

```suggestion
social-auth-core[azuread,openidconnect]==3.3.3
```

---

Updated.